x-pack/plugins/security_solution/public/app/search/index.ts

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { BehaviorSubject } from 'rxjs';
+
+import { AppUpdater } from 'src/core/public';
+import { LicenseType } from '../../../../licensing/common/types';
+import { SecuritySubPluginNames, SecurityDeepLinks } from '../types';
+import { AppMeta } from '../../../../../../src/core/public';
+
+const securityDeepLinks: SecurityDeepLinks = {
```

Comment:
If the paths used for the pages are changed, and these configurations aren't changed, that would introduce a bug. Perhaps you could write a functional test to prove that this works. The test could search for the pages in the global search bar and click the results (and verify that the expected page shows up.)

If nothing else, it would be good to add a similar test to a manual test plan.

Reply:
Yes, I agree. Will add a functional test later.